dashboard.json

```diff
@@ -15,6 +15,20 @@
       "type": "datasource",
       "pluginId": "fetzerch-sunandmoon-datasource",
       "pluginName": "Sun and Moon"
+    },
+    {
+      "name": "VAR_AVG_PER_KWH",
+      "type": "constant",
+      "label": "Cost per kWh",
+      "value": "0.19",
+      "description": ""
+    },
+    {
+      "name": "VAR_TZ",
+      "type": "constant",
+      "label": "Timezone",
+      "value": "America/Los_Angeles",
```

Comment:
The `tz.sh` script will still update this in the `dashboard.json` file to whatever the user enters during `setup.sh` so the input should default to the users preferred timezone.  That should be fine. I still like this as an input var.